nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,18 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nrf_modem 1.0.2
+***************
+
+* Implemented RAI (Release Assistance Indication) support in Modem library.
+* Fixed an issue that leads to the reporting of both POLLIN and POLLHUP by poll() when a connection is closed by the peer.
```

Comment:
This (and all other instances)  should be updated to reflect the actual name of the function/macro in the library to the doc, e.g. :c:func:\`nrf_poll\`, `NRF_POLLIN`; see the previous changelog entries.